packages/angular/cli/models/schematic-command.ts

```diff
@@ -322,6 +322,35 @@ export abstract class SchematicCommand<
             const validator = definition.validator;
             if (validator) {
               question.validate = input => validator(input);
+
+              // Filter allows transformation of the value prior to validation
+              question.filter = async (input) => {
+                for (const type of definition.propertyTypes) {
+                  let value;
+                  switch (type) {
+                    case 'string':
+                      value = String(input);
+                      break;
+                    case 'integer':
+                    case 'number':
+                      value = Number(input);
+                      break;
+                    case 'boolean':
```

Comment:
What is the use case of the boolean coercion here? `false` will be coerced to `true` in this case.

```
Boolean('false') // true
```